pages/philanthropy/index.js

```diff
@@ -298,37 +299,59 @@ const Philanthropy = ({ posts = [] }) => {
             >
               Investing in the future
             </Text>
+            <Fade bottom delay={90}>
+              <Button
+                variant="ctaLg"
+                my={3}
+                sx={{ width: ['100%', 'auto'] }}
+                as="a"
+                href="https://bank.hackclub.com/donations/start/hq"
+              >
+                Donate
+                <Text sx={{ display: ['none', 'inline-block'], ml: 2 }}>
+                  to Hack Club
+                </Text>
+              </Button>
+              <Text
+                sx={{ mt: 1, display: 'block', opacity: 0.8 }}
+                fontSize={2}
+                color="white"
+              >
+                Your contribution is tax-deductible.
+                <br />
+                Hack Club is a 501(c)(3) nonprofit with the EIN 81-2908499.
+              </Text>
+            </Fade>
```

Comment:
What if we adapted the header on the current `/donate`?

Reply:
omg duh! yes that's a great idea! thank you!